include/eve/module/core/regular/is_even.hpp

```diff
@@ -51,34 +51,32 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!      template< eve::value T >
-//!      eve::as_logical<T> is_even(T x) noexcept;
+//!      // Regular overload
+//!      constexpr auto is_even(value auto x) noexcept;                          // 1
+//!
+//!      // Lanes masking
+//!      constexpr auto is_even[conditional_expr auto c](value auto x) noexcept; // 2
+//!      constexpr auto is_even[logical_value auto m](value auto x) noexcept;    // 2
 //!   }
 //!   @endcode
 //!
 //!   **Parameters**
 //!
-//!     * `x` :  [argument](@ref eve::value).
+//!     * `x`:  [argument](@ref eve::value).
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!   **Return value**
 //!
-//!     The call `is_even(x)` is semantically  equivalent to:
-//!      @code
-//!      if constexpr(floating_value<T>)   return eve::is_flint (x) && eve::is_flint (x/2);
-//!      else constexpr(integral_value<T>) return eve::is_equal (x, (x/2)*2);
-//!      @endcode
+//!     1. The call `is_even(x)` is semantically  equivalent to:
+//!        @code
+//!        if constexpr(floating_value<T>)   return eve::is_flint (x) && eve::is_flint (x/2);
```

Comment:
I woudl prefer two separate bullet for each type. Code semantic is meh for those simple cases